src/app/component/MissionGroup/MissionGroup.jsx

```diff
@@ -55,6 +55,14 @@ const MissionGroup = ({ callToAction, group, groupCallToAction, showViewRoute })
   };
   const onClickViewRouteButton = () => {
     console.log("View Routes");
```

Comment:
If we're implementing this functionality on this callback, we're gonna need to remove this console log